components/add_groups_to_team_modal/add_groups_to_team_modal.tsx

```diff
@@ -129,16 +144,17 @@ export default class AddGroupsToTeamModal extends React.PureComponent {
         this.setState({saving: true});
 
         groupIDs.forEach(async (groupID) => {
-            const {error} = await this.props.actions.linkGroupSyncable(groupID, this.props.currentTeamId, Groups.SYNCABLE_TYPE_TEAM, {auto_add: true});
+            const {error} = await this.props.actions.linkGroupSyncable(groupID, this.props.currentTeamId, Groups.SYNCABLE_TYPE_TEAM, {auto_add: true, scheme_admin: false});
```

Comment:
I'm not sure if it not changes behaviour, but type requires it.
Related issue: https://github.com/mattermost/mattermost-redux/issues/1362

Reply:
that is a good question.

@nevyangelova could you have a look at it, since you are more proficient with redux?

Reply:
As far as I can see the scheme_admin prop is required by the server so it can pass validation and it looks like a recent add, do you have any concerns with passing that prop @komik966 ?

Reply:
@nevyangelova 
http body before my changes:
```
{ "auto_add": true }
```
http body after my changes:
```
{ "auto_add": true, "scheme_admin": false }
```
When body comes to the server, it goes through this path:

https://github.com/mattermost/mattermost-server/blob/8199e7b5f3f2e8eb8240a09f0fa47ee9d10a206d/api4/group.go#L32-L33

https://github.com/mattermost/mattermost-server/blob/8199e7b5f3f2e8eb8240a09f0fa47ee9d10a206d/api4/group.go#L242

https://github.com/mattermost/mattermost-server/blob/8199e7b5f3f2e8eb8240a09f0fa47ee9d10a206d/model/group_syncable.go#L131

My concern is if server treats `nil` same way as `false`.